internal/testutil/pkgdata_test.go

```diff
@@ -315,6 +315,21 @@ var pkgdataCheckEntries = []checkTarEntry{{
 		ModTime:  epochStartTime,
 		Format:   tar.FormatGNU,
 	},
+}, {
+	testutil.TarEntry{
+		Header: tar.Header{
+			Name: "",
```

Comment:
That feels like a strange case that makes me tempted to just complain about. That said, this is a test package, so I can imagine it being used precisely for testing, so sounds okay I guess.